internal/pubsub/samples/pub/main.go

```diff
@@ -0,0 +1,161 @@
+// pub is a sample application that publishes messages from stdin to an
+// existing topic.
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/google/subcommands"
+	"gocloud.dev/internal/pubsub"
+)
+
+type newCmd struct{}
+
+func (*newCmd) Name() string     { return "new" }
+func (*newCmd) Synopsis() string { return "Create a new topic" }
+func (*newCmd) Usage() string {
+	return `new <topic>:
+	Create a new topic.
+`
+}
+
+func (*newCmd) SetFlags(f *flag.FlagSet) {
+}
+
+func (c *newCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
+	if f.NArg() != 1 {
+		f.Usage()
+		return subcommands.ExitUsageError
+	}
+	topicID := f.Arg(0)
+	ctx := context.Background()
+	var err error
+	switch envFlag {
+	case "gcp":
+		err = makeGCPTopic(ctx, topicID)
+	case "rabbit":
+		err = makeRabbitTopic(topicID)
+	default:
+		log.Fatalf("unrecognized -env: %s", envFlag)
+	}
+	if err != nil {
+		log.Println(err)
+		return subcommands.ExitFailure
+	}
+	return subcommands.ExitSuccess
+}
+
+type delCmd struct{}
+
+func (*delCmd) Name() string     { return "del" }
```

Comment:
Why the `new` and `delete` commands? I buy a simple app that shows how to use the pubsub APIs to send to a topic and subscribe/tail a topic, but this functionality isn't in our API.

Reply:
Done. PTAL.